tests/ssl-opt.sh

```diff
@@ -941,6 +941,33 @@ run_test    "Default, DTLS" \
             -s "Protocol is DTLSv1.2" \
             -s "Ciphersuite is TLS-ECDHE-RSA-WITH-CHACHA20-POLY1305-SHA256"
 
+requires_config_enabled MBEDTLS_X509_TRUSTED_CERTIFICATE_CALLBACK
+run_test    "CA callback on client" \
+            "$P_SRV debug_level=3" \
+            "$P_CLI ca_callback=1 debug_level=3 " \
+            0 \
+            -c "use CA callback for X.509 CRT verification"\
+            -s "Protocol is TLSv1.2" \
```

Comment:
Minor: I think this line as well as the next three are out-of-place here. They make sense in the test they were copied from which is explicitly about testing defaults, but not here. They should probably be removed in order to avoid confusion and gain some flexibility (for example, the check for `ECDHE curve` would prevent this test from passing in a reduced configuration with only P-256 enable).

Reply:
Removed the lines